lab_7_llm/main.py

```diff
@@ -41,6 +48,9 @@ def obtain(self) -> None:
         Raises:
             TypeError: In case of downloaded dataset is not pd.DataFrame
         """
+        self._raw_data = (load_dataset('cointegrated/nli-rus-translated-v2021', split='dev')
```

Comment:
Use a variable  `self._hf_name` to pass the dataset name

Reply:
done